From e5a84190b4efc5e77f53277829a70a0ca7709f64 Mon Sep 17 00:00:00 2001
From: Daniel Ching <dching@nvidia.com>
Date: Wed, 16 Apr 2025 17:40:01 -0500
Subject: [PATCH 3/4] BLD: Use FOO_FOUND variables with find_package(FOO)

---
 cmake/Dependencies.cmake | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/cmake/Dependencies.cmake b/cmake/Dependencies.cmake
index 951fc69..00fb032 100644
--- a/cmake/Dependencies.cmake
+++ b/cmake/Dependencies.cmake
@@ -131,11 +131,11 @@ endif()
 set(TIFF_LIBRARY_DEPS)
 
 find_package(ZLIB)
-if(NOT DEFINED ZLIB_LIBRARY)
+if(NOT ZLIB_FOUND)
     message(STATUS "zlib not found - disabled")
 else()
-    message(STATUS "Using zlib at ${ZLIB_LIBRARY}")
-    list(APPEND TIFF_LIBRARY_DEPS ${ZLIB_LIBRARY})
+    message(STATUS "Using zlib at ${ZLIB_LIBRARIES}")
+    list(APPEND TIFF_LIBRARY_DEPS ${ZLIB_LIBRARIES})
 endif()
 
 find_package(ZSTD)
@@ -147,14 +147,14 @@ else()
 endif()
 
 find_package(JPEG 62) # 1.5.3 version
-if(NOT DEFINED JPEG_LIBRARY)
+if(NOT JPEG_FOUND)
     message(STATUS "libjpeg-turbo not found - disabled")
     set(BUILD_LIBJPEG_TURBO_EXT OFF CACHE BOOL INTERNAL)
     set(BUILD_LIBJPEG_TURBO_EXT OFF)
 else()
-    message(STATUS "Using libjpeg-turbo at ${JPEG_LIBRARY}")
-    include_directories(SYSTEM ${JPEG_INCLUDE_DIR})
-    list(APPEND TIFF_LIBRARY_DEPS ${JPEG_LIBRARY})
+    message(STATUS "Using libjpeg-turbo at ${JPEG_LIBRARIES}")
+    include_directories(SYSTEM ${JPEG_INCLUDE_DIRS})
+    list(APPEND TIFF_LIBRARY_DEPS ${JPEG_LIBRARIES})
 endif()
 
 find_package(TIFF)
-- 
2.49.0

